cwl/src/main/scala/wdl4s/cwl/package.scala

```diff
@@ -52,6 +33,8 @@ package object cwl extends TypeAliases {
 
   type Cwl = Workflow :+: CommandLineTool :+: CNil
 
+  type Checked[A] = Either[NonEmptyList[String], A]
```

Comment:
I think it'd be cool if this was added in `lenthall` alongside some helper extensions (eg `toErrorOr`?) if we can start using it elsewhere too?

Reply:
Yes it should be.  I'll create a PR